### PR TITLE
Fix solution files via slngen

### LINKED ISSUE #72052: [JSInterop] cannot build System.Net.Http.sln
Probably fallout from https://github.com/dotnet/runtime/pull/66304

```
error : The project file could not be loaded. Could not find a part of the path '/home/manicka/repositories/runtime/src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System.Private.Runtime.InteropServices.JavaScript.csproj'.  /home/manicka/repositories/runtime/src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System.Private.Runtime.InteropServices.JavaScript.csproj [/home/manicka/repositories/runtime/src/libraries/System.Net.Http/System.Net.Http.sln]
```

cc @pavelsavara

### PR DESCRIPTION
**Pull request by pavelsavara**
Fixes https://github.com/dotnet/runtime/issues/72052

### CODE REVIEW

**Comment by pavelsavara** [comment]
Should the solutions like HTTP have dependency on the new `System.Runtime.InteropServices.JavaScript` and `JSImportGenerator` assemblies ? 
They are used only when building for Browser target and that never works for me in the VS anyway.

**Comment by ViktorHofer** [comment]
> Should the solutions like HTTP have dependency on the new System.Runtime.InteropServices.JavaScript and JSImportGenerator assemblies ?
They are used only when building for Browser target and that never works for me in the VS anyway.

Solution files are auto-generated via the slngen tooling which automatically detects which projects need to be inside a solution. Please run `dotnet.cmd src/libraries/slngen.proj` to re-generate all solution files. There might be a few unrelated changes but that's OK.

File: src/libraries/System.Private.Runtime.InteropServices.JavaScript/System.Private.Runtime.InteropServices.JavaScript.sln [hunk 1]
```diff
@@ -1,8 +1,10 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32602.215
+MinimumVisualStudioVersion = 10.0.40219.1
```
Comment (ViktorHofer) [inline src/libraries/System.Private.Runtime.InteropServices.JavaScript/System.Private.Runtime.InteropServices.JavaScript.sln:5]: We usually don't check in these headers. As mentioned, please re-generate all solution files via slngen.

**Comment by pavelsavara** [comment]
Thanks, I didn't know we have that tool!

> There might be a few unrelated changes but that's OK.

There are quite "few" :-)

**Review by ViktorHofer — APPROVED** [review]

**Comment by ViktorHofer** [comment]
19 is actually not that much :) Thanks for updating.

**Pull request merged**